src/Microsoft.Restier.Core/Query/QueryExpressionContext.cs

```diff
@@ -174,6 +174,9 @@ private QueryModelReference ComputeModelReference()
             QueryModelReference modelReference = null;
 
             var methodCall = this.VisitedNode as MethodCallExpression;
+            var parameter = this.VisitedNode as ParameterExpression;
+            var member = this.VisitedNode as MemberExpression;
```

Comment:
Two options,
1. like what I do now.
2. Any if condition is not null, return
Which one you prefer? 

Previous logic does not return, it will go through all logic and last return.